pkg/ingress/controller/controller.go

```diff
@@ -108,6 +108,22 @@ const (
 	// IngressControllerTag is added to the related resources.
 	IngressControllerTag = "octavia.ingress.kubernetes.io"
 
+	// IngressAnnotationTimeoutClientData is the timeout for client data in ms. Defaults to 50000
+	// Refer https://docs.openstack.org/octavia/latest/configuration/configref.html
```

Comment:
Adding a link is a nice touch!
I would also mention the exact configuration switch though.

```suggestion
	// IngressAnnotationTimeoutClientData is the timeout for frontend client inactivity.
	// If not set, this value defaults to the Octavia configuration key `timeout_client_data`.  
	// Refer to https://docs.openstack.org/octavia/latest/configuration/configref.html#haproxy_amphora.timeout_client_data
```
timeout_client_data